aot/src/test/resources/com/dylibso/chicory/approvals/ApprovalTest.functions10.approved.txt

```diff
@@ -110,13 +105,496 @@ public final class com/dylibso/chicory/$gen/CompiledMachine implements com/dylib
     ICONST_0
     ILOAD 6
     ALOAD 7
-    INVOKESTATIC com/dylibso/chicory/$gen/CompiledMachine$AotMethods.callIndirect ([JIILcom/dylibso/chicory/runtime/Instance;)[J
+    INVOKESTATIC com/dylibso/chicory/$gen/CompiledMachineAotMethods.callIndirect ([JIILcom/dylibso/chicory/runtime/Instance;)[J
     ICONST_0
     LALOAD
     L2I
     IRETURN
 }
 
+final class com/dylibso/chicory/$gen/CompiledMachineAotMethods {
```

Comment:
nitpick: we should keep `CompiledMachineAotMethods` out of the approvals I guess. but we can fix in #871 if you think that's ok for this iteration